queries.sql

```diff
@@ -8,3 +9,55 @@ SELECT name , escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*Day Two */
+
+/* Transaction - BEGIN, UPDATE and COMMIT */
+
+/*UPDATE*/
+BEGIN TRANSACTION;
+UPDATE animals SET species='unspecified';
+
+SELECT species FROM animals;
+ROLLBACK TRANSACTION;
+SELECT species FROM animals;
+
+UPDATE animals SET species='digimon'
+	WHERE name::text LIKE '%mon';
+	
+UPDATE animals SET species='pokemon'
+	WHERE species ISNULL;
+	
+COMMIT TRANSACTION;
+
+SELECT * FROM animals ;
+
```

Comment:
- [ ] This is a separate transaction so I'd advise you to add a `BEGIN` statement before line `25` to mark the start of the transaction.